keyboards/handwired/dactyl_manuform/6x7/config.h

```diff
@@ -26,9 +26,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define MATRIX_COLS 7
 
 // wiring of each half
-#define MATRIX_COL_PINS { D4, C6, D7, E6, B4, B5, B7 }
+#define MATRIX_COL_PINS { D4, C6, D7, E6, B4, B5, F4 }
```

Comment:
```suggestion
#define MATRIX_COL_PINS { D4, C6, D7, E6, B4, B5, F4 }
```
Why do you see this change necessary? This will break compatibility for anyone using the current 6x7 firmware.

Reply:
Specifically, your keymap's config.h should use: 
```c
#undef MATRIX_COL_PINS
#define MATRIX_COL_PINS { D4, C6, D7, E6, B4, B5, F4 }
```